packages/auth-remix/package.json

```diff
@@ -16,18 +16,21 @@
   ],
   "scripts": {
     "typecheck": "tsc --project tsconfig.json --noEmit",
-    "build": "tsc --project tsconfig.json"
+    "build": "tsc --project tsconfig.json",
+    "lint": "eslint --quiet",
+    "lint:fix": "eslint --fix"
   },
   "devDependencies": {
     "@types/node": "^20.12.13",
-    "edgedb": "^1.5.0",
-    "typescript": "^5.5.2"
+    "typescript": "^5.5.2",
+    "edgedb": "*",
+    "@repo/tsconfig": "*"
   },
   "peerDependencies": {
     "edgedb": "^1.3.6"
   },
   "dependencies": {
-    "@edgedb/auth-core": "^0.2.0",
+    "@edgedb/auth-core": "*",
```

Comment:
Can you remind me how this works when we actually publish this package? This gets replaced by a specific version, right?

Reply:
Ahh, right. I was confusing this with the `workspace:` protocol syntax which does some of this publish-time magical replacement: https://yarnpkg.com/features/workspaces#cross-references

Yeah, let's prefer specific versions until we have something like that setup.